scripts/js/lib/api/updateLinks.ts

```diff
@@ -62,7 +62,10 @@ export function normalizeUrl(
   kwargs: { kebabCaseAndShorten: boolean; pkgName: string },
 ): string {
   if (isAbsoluteUrl(url)) return url;
-  if (url.startsWith("/")) return url;
+  // We only want to normalize the API URLs to transform them to kebab-case
+  // Todo: update URLs from other APIs, when they all use kebab-case.
+  if (url.startsWith("/") && !url.startsWith(`/api/${kwargs.pkgName}`))
+    return url;
```

Comment:
I don't think I understand this? Your PR comment makes it sound necessary and good, but I don't think I understand why we need this? Notably, a lot of the URLs we do normalize are links to the API docs!

Is this maybe more accurate?

> If the URL is already pointing to `/api`, we assume that it was already normalized. This can happen, for example, when .... We need to be careful to ... because otherwise ...

Please make sure @frankharkins understands very well why we need this code. A good test is for him to explain to you why we can't delete that code, based only on him reading this section of the code (not your PR review).